crates/router/src/routes/app.rs

```diff
@@ -1776,6 +1776,10 @@ impl Profile {
                             web::resource("/toggle")
                                 .route(web::post().to(routing::toggle_elimination_routing)),
                         ),
+                    )
+                    .service(
+                        web::resource("/set_volume_split")
```

Comment:
By just looking at the api-contract, client won't be aware that he is setting volume split for dynamic routing type right? Even the query param is just `split`

Reply:
this is scoped under `/dynamic_routing`